docs/zh_CN/Compression/AutoPruningUsingTuners.rst

```diff
@@ -64,7 +64,7 @@ op_type 'Conv2d' 表示在 PyTorch 框架下定义在 :githublink:`default_layer
     trialConcurrency: 1
     trialGpuNumber: 0
     tuner:
-      name: grid
+      name: GridSearch
```

Comment:
The Chinese doc is auto-translated from the English doc, directly modify will meet an error, please revert it.

Reply:
OK